tool/generate_converter

```diff
@@ -65,7 +65,18 @@ for type in $(echo "$source" \
         continue
       fi
       field_type=$(echo $line | sed -e 's#  final ##' | sed -e 's# .*##')
-      if echo "$field_type" | egrep -q '^(Argument|Element|Expression|RecordField|Reference|StringLiteralPart)$'; then
+      if echo "$field_type" | grep -E -q '^(Argument|Element|Expression|RecordField|Reference|StringLiteralPart)$'; then
+        echo "    $field_name: convertTo$field_type(o.$field_name),"
+      else
+        echo "    $field_name: convert(o.$field_name),"
+      fi
+    elif echo "$line" | grep -q ' get '; then
```

Comment:
name is defined as a getter for expressions, hence the new if branch

https://github.com/dart-lang/sdk/blob/805935df2102a8506d51c6ff0119a5b0d69074d4/pkg/_fe_analyzer_shared/lib/src/metadata/references.dart#L12